pom.xml

```diff
@@ -16,25 +16,27 @@
     <description>API for document OCR</description>
 
     <properties>
-        <java.version>11</java.version>
+        <java.version>21</java.version>
 
         <included.tests>unit-test</included.tests>
 
         <apache-commons-lang.version>2.6</apache-commons-lang.version>
         <environment-reader.version>1.3.10</environment-reader.version>
-        <structured-logging.version>1.9.26</structured-logging.version>
+        <structured-logging.version>3.0.3</structured-logging.version>
+        <!-- Needed for structured-logging -->
```

Comment:
I would remove this comment. Is it not a direct dependency because it is used by LoggingInterceptor?

Reply:
thanks @rafahop I have removed it.